bin/webpack.js

```diff
@@ -378,6 +384,10 @@ yargs.parse(process.argv.slice(2), (err, argv, output) => {
 			}
 		});
 
+		ifArg("info-verbosity", function(value) {
+			outputOptions.infoVerbosity = value;
```

Comment:
You don't have to put this into `outputOptions`. A local variable would be fine.